### DISCUSSION TITLE
IC40 Intellchlor Issue

### DISCUSSION THREAD

**Discussion by LTrainExpress** [tagyoureit/nodejs-poolController#478] [RS485] [answered]
I added my IC40 to njsPC, but it doesn't seem to be talking. I checked and double checked the RS485 wiring. This is what my config looks like.

![image](https://user-images.githubusercontent.com/15890941/172614852-2e29c195-2a0b-45b9-8d3a-95fa3f162a28.png)
![image](https://user-images.githubusercontent.com/15890941/172614906-460f65eb-1343-4e00-8257-cb6286e86008.png)

I also performed a capture of all my equipment in replay-all and a capture of just broadcast/chlorinator/invalid in replay-chlor. Any ideas why no data is showing up?

[replay-all.zip](https://github.com/tagyoureit/nodejs-poolController/files/8861468/replay-all.zip)
[replay-chlor.zip](https://github.com/tagyoureit/nodejs-poolController/files/8861470/replay-chlor.zip)

**Comment by rstrouse** [comment, marked as answer]
You created this as a chlorinator that is attached to an external controller.  Delete the chlorinator and re-create it.  Select Nixie as the master on the controller selection dialog.
